open-notes/maturity-model.md

```diff
@@ -0,0 +1,394 @@
+
+### Level 7
+
+- [ ] [Honest types](#honest-types) [code, knowledge]
+- [ ] [Pair programming](#pair-programming) [process, knowledge]
+- [ ] [Correctness alerts](#correctness-alerts) [observability]
+- [ ] [APM tracing](#apm-tracing) [observability]
+- [ ] [Resilient requests](#resilient-requests) [code]
+
+### Level 8
+
+- [ ] [Embrace immutable code](#embrace-immutable-code) [code]
+- [ ] [Mob programming](#mob-programming) [process, knowledge, communication]
+- [ ] [Learning days](#learning-days) [process, knowledge]
+- [ ] [Synthetics](#sythetics) [observability]
+
+### Level 9
+
+- [ ] [Cyclomatic complexity ceiling](#cyclomatic-complexity-ceiling) [code, observability]
+- [ ] [Quantitative team health](#quantitative-team-health) [observability, process]
+- [ ] [Embrace FP principles](#embrace-fp-principles) [code]
+- [ ] [PR to production](#pr-to-production) [process]
+
+### Level 10
+
+- [ ] [Dashboards IV](#dashboards-iv) [observability, process]
+- [ ] [Load testing](#load-testing) [maintenance]
+- [ ] [Immutable Architecture](#immutable-architecture) [code]
+
+### Level 11
+
+- [ ] [Domain Driven design](#domain-driven-design) [process, knowledge, code, communication]
+- [ ] [Property based testing](#property-based-testing) [maintenance]
+- [ ] [Chaos engineering](#chaos-engineering) [code, maintenance]
+
+
+## Skills
+
+### Unit testing
+
+Unit testing are written for all new features. The unit tests are readable, and [easy to maintain](https://devonburriss.me/maintainable-unit-tests/) using an ever growing set of helpers designed specifically for that domain. Unit tests are submitted with the new code or changes they are covered, thus actually ensuring some quality before code is deployed to production.
+
+### Centralized and prioritized stories
+
+The team keeps a single shared list of work that is being worked on and the list is prioritized from highest priority to lowest... preferably based on value, not urgency.
+
+### Regular team retrospective
+
+The team has a regular moment, AT MINIMUM monthly, to reflect on how things have gone over the last while and suggest improvements, which they then implement. Hopefully one of the first things to come out is monthly is too long a time to reflect on possible improvements.
+
+### Centralized Structured Logging
+
+Simply logging is not enough. Logs should be collected in a central place where they can be easily searched. To facilitate the searching, logs should be structured, rather than free text logs.
+
+### No secrets in source control
+
+No secrets to any environments should be in source control. Not just the current HEAD but any previous secrets should be scrubbed from history with something like [BFG](https://rtyley.github.io/bfg-repo-cleaner/).
+
+### Deploy monthly
+
+The minimum is that each application is deployed at least once per month. This includes applications that have not changed in that month to ensure that pipelines are being maintained.
```

Comment:
I see two principles here: changed and unchanged code. Are you not trying to achieve different goals?